Document Wallets Service Request/Response Models (#579)
trinsic-id/sdk: docs/reference/services/wallet-service.md

## Create Wallet

Wallets can be created directly by the user or through an invitation by the ecosystem provider. Depending on the ecosystem settings, direct wallet creation may not be enabled for your provider. The wallet is created automatically upon user signin. For more information on that, see the link below:

- [Account Sign In](./account-service.md#sign-in)

## Insert Item

> Trinsic supports the ability to insert verifiable credentials in a wallet simply using JSON data. The method below illustrate how that can be achieved.

Reviewer: - I'd remove the second sentence here -- the following code block will, in my mind, be clearly linked to this operation already.
  - If the sentence stays, `illustrate` should be changed to `illustrates`, and `that` should maybe be changed to `this`.
- `in a wallet` should maybe be changed to `into a wallet`.

Reviewer 2: @geel9 You can also suggest edits through the review comment using the `Add Suggestion` button. This will give the author one button to `Accept All Suggestion` and amend a commit to the stream. It's super convenient for suggestions like this

> Trinsic supports the ability to insert verifiable credentials in a wallet simply using JSON data. The method below illustrate how that can be achieved.

Reviewer: ```suggestion
Trinsic supports the ability to insert verifiable credentials into a wallet simply using JSON data.
```
